releases/mainnet/1.9.4/contracts.json

```diff
@@ -0,0 +1,252 @@
+  "MCD_CLIP_CALC_USDC_A": "0x0FCa4ba0B80123b5d22dD3C8BF595F3E561d594D",
+  "MCD_JOIN_USDC_B": "0x2600004fd1585f7270756DDc88aD9cfA10dD0428",
+  "MCD_CLIP_USDC_B": "0x5590F23358Fe17361d7E4E4f91219145D8cCfCb3",
+  "MCD_CLIP_CALC_USDC_B": "0xD6FE411284b92d309F79e502Dd905D7A3b02F561",
+  "MCD_JOIN_PSM_USDC_A": "0x0A59649758aa4d66E25f08Dd01271e891fe52199",
+  "MCD_CLIP_PSM_USDC_A": "0x66609b4799fd7cE12BA799AD01094aBD13d5014D",
+  "MCD_CLIP_CALC_PSM_USDC_A": "0xbeE028b5Fa9eb0aDAC5eeF7E5B13383172b91A4E",
+  "MCD_PSM_USDC_A": "0x89B78CfA322F6C5dE0aBcEecab66Aee45393cC5A",
+  "MCD_JOIN_PSM_PAX_A": "0x7bbd8cA5e413bCa521C2c80D8d1908616894Cf21",
+  "MCD_CLIP_PSM_PAX_A": "0x5322a3551bc6a1b39d5D142e5e38Dc5B4bc5B3d2",
+  "MCD_CLIP_CALC_PSM_PAX_A": "0xC19eAc21A4FccdD30812F5fF5FebFbD6817b7593",
+  "MCD_PSM_PAX_A": "0x961Ae24a1Ceba861D1FDf723794f6024Dc5485Cf",
+  "PIP_PSM_PAX": "0x043B963E1B2214eC90046167Ea29C2c8bDD7c0eC",
+  "WBTC": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
+  "PIP_WBTC": "0xf185d0682d50819263941e5f4EacC763CC5C6C42",
+  "MCD_JOIN_WBTC_A": "0xBF72Da2Bd84c5170618Fbe5914B0ECA9638d5eb5",
+  "MCD_CLIP_WBTC_A": "0x0227b54AdbFAEec5f1eD1dFa11f54dcff9076e2C",
+  "MCD_CLIP_CALC_WBTC_A": "0x5f4CEa97ca1030C6Bd38429c8a0De7Cd4981C70A",
+  "TUSD": "0x0000000000085d4780B73119b644AE5ecd22b376",
+  "PIP_TUSD": "0xeE13831ca96d191B688A670D47173694ba98f1e5",
+  "MCD_JOIN_TUSD_A": "0x4454aF7C8bb9463203b66C816220D41ED7837f44",
+  "MCD_CLIP_TUSD_A": "0x0F6f88f8A4b918584E3539182793a0C276097f44",
+  "MCD_CLIP_CALC_TUSD_A": "0x059acdf311E38aAF77139638228d393Ff27639bF",
+  "ZRX": "0xE41d2489571d322189246DaFA5ebDe1F4699F498",
+  "PIP_ZRX": "0x7382c066801E7Acb2299aC8562847B9883f5CD3c",
+  "MCD_JOIN_ZRX_A": "0xc7e8Cd72BDEe38865b4F5615956eF47ce1a7e5D0",
+  "MCD_CLIP_ZRX_A": "0xdc90d461E148552387f3aB3EBEE0Bdc58Aa16375",
+  "MCD_CLIP_CALC_ZRX_A": "0xebe5e9D77b9DBBA8907A197f4c2aB00A81fb0C4e",
+  "KNC": "0xdd974D5C2e2928deA5F71b9825b8b646686BD200",
+  "PIP_KNC": "0xf36B79BD4C0904A5F350F1e4f776B81208c13069",
+  "MCD_JOIN_KNC_A": "0x475F1a89C1ED844A08E8f6C50A00228b5E59E4A9",
+  "MCD_CLIP_KNC_A": "0x006Aa3eB5E666D8E006aa647D4afAB212555Ddea",
+  "MCD_CLIP_CALC_KNC_A": "0x82c41e2ADE28C066a5D3A1E3f5B444a4075C1584",
+  "MANA": "0x0F5D2fB29fb7d3CFeE444a200298f468908cC942",
+  "PIP_MANA": "0x8067259EA630601f319FccE477977E55C6078C13",
+  "MCD_JOIN_MANA_A": "0xA6EA3b9C04b8a38Ff5e224E7c3D6937ca44C0ef9",
+  "MCD_CLIP_MANA_A": "0xF5C8176E1eB0915359E46DEd16E52C071Bb435c0",
+  "MCD_CLIP_CALC_MANA_A": "0xABbCd14FeDbb2D39038327055D9e615e178Fd64D",
+  "USDT": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+  "PIP_USDT": "0x7a5918670B0C390aD25f7beE908c1ACc2d314A3C",
+  "MCD_JOIN_USDT_A": "0x0Ac6A1D74E84C2dF9063bDDc31699FF2a2BB22A2",
+  "MCD_CLIP_USDT_A": "0xFC9D6Dd08BEE324A5A8B557d2854B9c36c2AeC5d",
+  "MCD_CLIP_CALC_USDT_A": "0x1Cf3DE6D570291CDB88229E70037d1705d5be748",
+  "PAXUSD": "0x8E870D67F660D95d5be530380D0eC0bd388289E1",
+  "PIP_PAXUSD": "0x043B963E1B2214eC90046167Ea29C2c8bDD7c0eC",
```

Comment:
Did we rename `PAXUSD` and `PIP_PAXUSD` in the Chainlog?
If `PAXUSD-A` is gonna be deprecated (if not already), then we should rename these entries to just `PAX` and `PIP_PAX` and I think the PSM should be in this section, right after `PAXUSD_A` (to follow the same than USDC).
If we didn't do the renaming in the Chainlog, then we should keep them as they are right now here and in the next spell we can fix that and change it here.

Reply:
We did not update the changelog address for PAX in the most recent executive. I will add a card to do add this in a future exec. In the meantime, we should keep `PAXUSD` since that's what matches what's in the onchain changelog.

Reply:
Moved the pax addresses at b8797cf57d4fd0f5358e522fb815d0843b81c6c4